sdk/communication/communication-chat/test/chatClient.spec.ts

```diff
@@ -86,19 +86,14 @@ describe("ChatClient", function() {
       threadId = chatThreadResult.chatThread?.id!;
 
       // Create ChatThreadClient
-      chatThreadClient = await chatClient.getChatThreadClient(threadId);
```

Comment:
Was the await removed on purpose?

Reply:
yup, it was redundant because getChatThreadClient is not async